src/app/clinic-dashboard/hiv/hiv-lists-microfrontend-report/hiv-lists-microfrontend.component.ts

```diff
@@ -43,12 +43,12 @@ export class HIVListsMicroFrontendComponent implements OnDestroy, OnInit {
       this.locationUuid = params.location_uuid;
     });
     this.url = this.sanitizer.bypassSecurityTrustResourceUrl(
-      `${this.baseUrl}/amrs/spa/home`
+      `${this.baseUrl}/openmrs/spa/home`
```

Comment:
Should this be amrs or openmrs since our deployment base is amrs

Reply:
nice catch removing it